### PR TITLE
[feature/cci-cleanup] 📦 🛢 Various cleanup

### PR DESCRIPTION
**Pull request by davisagli**
- Use jwt_session from CumulusCI
- Stop storing login_url since jwt_session can now derive it from instance_url
- Use task from CumulusCI for deploying org settings
- Fix get_unsaved_changes so that it doesn't update latest_revision_numbers

### CODE REVIEW

**Comment by davisagli** [comment]
@wlonk ok, got the builds to run

File: metashare/api/tests/jobs.py [hunk 1]
```diff
@@ -110,11 +110,11 @@ def test_get_unsaved_changes(scratch_org_factory):
...
 
-        assert scratch_org.unsaved_changes
-        assert scratch_org.latest_revision_numbers == {
-            "TypeOne": {"NameOne": 13},
-            "TypeTwo": {"NameTwo": 10},
+        assert scratch_org.unsaved_changes == {
+            "TypeOne": ["NameOne"],
+            "TypeTwo": ["NameTwo"],
         }
+        assert scratch_org.latest_revision_numbers == {"TypeOne:NameOne": 10}
```
Comment (wlonk) [inline metashare/api/tests/jobs.py]: We're not using this format anymore; if you look at what `get_latest_revision_numbers` returns, it's `Dict<str, Dict<str, int>>`.

Comment (davisagli) [reply metashare/api/tests/jobs.py]: I'm fighting a fire elsewhere; would you mind fixing? I just intended to assert that it's the same as what it was initialized to at the beginning of the test.

Comment (wlonk) [reply metashare/api/tests/jobs.py]: Yeah, I'm running this through functional tests locally to make sure I don't miss anything, and I'll push some fixes.

**Pull request merged**